images/base/files/usr/local/bin/entrypoint

```diff
@@ -203,18 +203,29 @@ fix_cgroup() {
     return
   fi
   echo 'INFO: detected cgroup v1'
-  echo 'INFO: fix cgroup mounts for all subsystems'
+  local current_cgroup
+  current_cgroup=$(grep -E '^[^:]*:([^:]*,)?cpu(,[^,:]*)?:.*' /proc/self/cgroup | cut -d: -f3)
+  if [ "$current_cgroup" = "/" ]; then
+    echo "INFO: cgroupns detected, no need to fix cgroups"
+    return
+  fi
+
+  echo 'WARN: cgroupns not enabled! Please use cgroup v2, or cgroup v1 with cgroupns enabled.'
+  echo 'WARN: Support for host cgroup namespace will be removed!'
```

Comment:
I'm not sure we're actually going to do that, hacky as it is.
The line above is good, I think this is my only nit with this PR.

Reply:
My thinking was that such a message should encourage more users to enable cgroupns, and not force us  to actually do anything (as the message does not specify when such support will be removed).

Anyway, message removed, and I rewrote the TODO item into the NOTE.

PR updated, PTAL @BenTheElder

Reply:
+1

BTW, For the record: Kubernetes's CI is unfortunately one of these places still.